src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/DaysOnLevelTest.kt

```diff
@@ -117,9 +117,9 @@ class DaysOnLevelTest {
       val iepTime = LocalDateTime.now().minusDays(10)
       val iepSummary = IepSummary(
         bookingId = 1L,
-        iepDate = iepTime.toLocalDate(),
+        iepDate = iepTime.toLocalDate().plusDays(9),
```

Comment:
I'm almost tempted to suggest building the list of `IepDetail` before the `IepSummary` and use `iepDetails[0].iepDate` to make this relationship even more explicit...

but I'm also happy with this :)